src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial.cpp

```diff
@@ -69,15 +68,19 @@ SF45LaserSerial::SF45LaserSerial(const char *port, uint8_t rotation) :
 	}
 
 	_num_retries = 2;
-	_px4_rangefinder.set_device_id(device_id.devid);
-	_px4_rangefinder.set_device_type(DRV_DIST_DEVTYPE_LIGHTWARE_LASER);
 
 	// populate obstacle map members
 	_obstacle_map_msg.frame = obstacle_distance_s::MAV_FRAME_BODY_FRD;
-	_obstacle_map_msg.increment = 5;
-	_obstacle_map_msg.angle_offset = 2.5;
-	_obstacle_map_msg.min_distance = UINT16_MAX;
+	_obstacle_map_msg.sensor_type = obstacle_distance_s::MAV_DISTANCE_SENSOR_LASER;
+	_obstacle_map_msg.increment = 10;
```

Comment:
There's some confusion as to how the resolution of collision prevention can be used. I think in collision prevention code it's assumed to be an increment of 10, however the obstacle distance array can support an increment as low as 5 deg for the "resolution", which works out to 360 degree resolution coverage wrapped in 72 bins (should be default imo). 

This was probably done this way since the original collision prevention impl leveraged point-lidars and the point-sample resolution was in the 10's of degrees: https://discuss.px4.io/t/question-on-distances-array-in-obstacle-distance/12911

In the case of the SF45: https://support.lightware.co.za/sf45b/#/specs

Reply:
Yea, as we discussed in the CP PR, let's switch here to 5° Degrees.
I also discussed with Matthias yesterday that if computation is not the limitation, it should also be possible to decrease the internal resolution to 1 deg, then we could publish distance_sensor messages with orientation set via the quaternion which would result in even better resolution. Maybe something to consider in the future if there is need for that.